Sources/Defaults/Utilities.swift

```diff
@@ -146,3 +144,58 @@ extension DispatchQueue {
 		}
 	}
 }
+
+public protocol NativelySupportedType {
+	associatedtype Property: NativelySupportedType = Self
+}
+
+extension Optional: NativelySupportedType where Wrapped: NativelySupportedType {
+	public typealias Property = Wrapped
+}
+
+extension Array: NativelySupportedType where Element: NativelySupportedType {
+	public typealias Property = Element
+}
+
+extension Dictionary: NativelySupportedType where Key == String, Value: DefaultsSerializable {
+	public typealias Property = Value
+}
+
+public protocol DefaultsSerializable: NativelySupportedType {
+	typealias Value = Bridge.Value
+	typealias Serializable = Bridge.Serializable
+	associatedtype Bridge: DefaultsBridge
+	static var bridge: Bridge { get }
+}
+
+extension Data: DefaultsSerializable {}
+extension Date: DefaultsSerializable {}
+extension Bool: DefaultsSerializable {}
+extension Int: DefaultsSerializable {}
+extension Double: DefaultsSerializable {}
+extension Float: DefaultsSerializable {}
+extension String: DefaultsSerializable {}
+
+extension Optional: DefaultsSerializable where Wrapped: DefaultsSerializable {
+	public static var bridge: DefaultsOptionalBridge<Wrapped.Bridge> { return DefaultsOptionalBridge(bridge: Wrapped.bridge) }
+}
+
+extension DefaultsSerializable where Self: Codable {
+	public static var bridge: DefaultsCodableBridge<Self> { return DefaultsCodableBridge<Self>() }
+}
+
+extension DefaultsSerializable where Self: RawRepresentable {
+	public static var bridge: DefaultsRawRepresentableBridge<Self> { return DefaultsRawRepresentableBridge() }
+}
+
+extension URL: DefaultsSerializable {
+	public static var bridge: DefaultsURLBridge { return DefaultsURLBridge() }
```

Comment:
```suggestion
	public static let bridge = DefaultsURLBridge()
```

Applies to the others too.